glide-core/src/client/value_conversion.rs

```diff
@@ -1023,6 +1255,16 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
                 None
             }
         }
+        b"XINFO STREAM" => {
+            if cmd.position(b"FULL").is_some() {
+                Some(ExpectedReturnType::XInfoStreamFullReturnType)
+            } else {
+                Some(ExpectedReturnType::Map {
+                    key_type: &Some(ExpectedReturnType::BulkString),
```

Comment:
As you mentioned, this change will mean the keys in the map response will be regular strings instead of byte strings. To me it looks like all the other commands return byte strings instead of regular strings. I wonder if it will be unexpected to get back some regular strings from this specific command? Especially since the other XINFO commands also return byte strings for the keys of the map